cropduster/views/__init__.py

```diff
@@ -296,7 +303,7 @@ def fit_preview(im):
 
     img = PIL.Image.open(cropduster_image.image.path)
     preview_file_path = cropduster_image.get_image_path('_preview')
-    if not os.path.exists(preview_file_path):
+    if not default_storage.exists(preview_file_path):
```

Comment:
I imagine that you'll need to change `Image.get_image_path` so that it ends with `return converted.name` rather than `converted.path`.